.ruby-version

```diff
@@ -1 +1 @@
-2.7.8
+2.7.7
```

Comment:
Why downgrade the Ruby version? Is there something in 2.7.8 that breaks in the new setting?

Reply:
Using 2.7.8, I get an error when running `bundle update`.
```
LoadError: incompatible library version - /Users/jblogs/docs/vendor/bundle/ruby/2.7.0/gems/strscan-3.1.0/lib/strscan.bundle
```
It may be an Apple Silicon issue but I don't know enough about it to diagnose and fix.

Reply:
Right, ok. There are two issues, I think:

- you shouldn't need to run `bundle update` -- that will try to get new versions of everything and update the lock file, which we do sometimes do, but only to explicitly update dependencies
- the `Makefile` will run `bundle install` which should install what is in `Gemfile.lock`, it's Ok to run that one manually as well
- `bundle install` may fail if you already have gems/libraries installed in the `vendor` directory from another version (either ruby version or from something new that `bundle update` brought in). My first try if you get that error message above would be to delete the `vendor` directory and retry `bundle install`.

I'll have a go at checking if that works on M1 and intel on the branch.

Reply:
I deleted the ```vendor``` directory and now I can run 2.7.8. Thanks!

Reply:
Cool! I'll drop the commits for the ruby version then.